lightly_studio/tests/resolvers/video_resolver/test_create_many.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from sqlmodel import Session
+
+from lightly_studio.models.dataset import SampleType
+from lightly_studio.models.video import VideoCreate
+from lightly_studio.resolvers import (
+    video_resolver,
+)
+from tests.helpers_resolvers import (
+    create_dataset,
+)
+
+
+class TestCreateMany:
```

Comment:
Do not nest it in a class. We nest in a class only if the tested functions are in a class.